web-app/js/portal/ui/Viewport.js

```diff
@@ -7,6 +7,21 @@
 
 Ext.namespace('Portal.ui');
 
+window.onbeforeunload = function() {
+
+    if (   Portal.config.isFeatureToggleEnabled('chatOnFailedSearch')
```

Comment:
Scratch that, then I'm fine with that. I do it myself quite a lot, was just a bit weird in the github viewer...